cli/sawtooth_cli/cluster.py

```diff
@@ -305,34 +295,13 @@ def do_cluster_stop(args):
     else:
         node_names = vnm.get_node_names()
 
-    state_nodes = state["Nodes"]
-
     # if node_names is empty, stop doesn't get called
     for node_name in node_names:
-        if node_name not in state_nodes:
-            raise CliException(
-                "{} is not a known node name".format(node_name))
-        if state_nodes[node_name]['Status'] == 'Stopped':
-            raise CliException('{} already stopped'.format(node_name))
 
         node_command_generator.stop(node_name)
 
-        # Update status of Nodes
-        node_status = 'Stopped' if node_name in state_nodes else 'Unknown'
-        state_nodes[node_name]['Status'] = node_status
-
-    if len(args.node_names) == 0 and len(node_names) == 0:
-        for node_name in state_nodes:
-            state_nodes[node_name]["Status"] = "Unknown"
-
-    # If none of the nodes are running set overall State to Stopped
-    state["DesiredState"] = "Stopped"
-    for node in state_nodes:
-        if state_nodes[node]["Status"] == "Running":
-            state["DesiredState"] = "Running"
+    state["DesiredState"] = "Stopped"  # Stop using this in state file
```

Comment:
Again, my oversight  - I decided we may want to leave DesiredState in the state file, as there is no other way to determine if a process running subprocess has died. I'll remove comment.